src/fragments/lib/reactnative.mdx

```diff
@@ -0,0 +1,16 @@
+## Amplify JavaScript
```

Comment:
Should this file be updated to reflect React Native? I think at the moment this is similar to what we have for Javascript

Reply:
Kept this the same since technically it is still using the Javascript library.